src/test/addrman_tests.cpp

```diff
@@ -92,22 +92,22 @@ BOOST_AUTO_TEST_CASE(addrman_simple)
     BOOST_CHECK(addr_null.ToString() == "[::]:0");
 
     // Test 2: Does Addrman::Add work as expected.
-    CService addr1 = ResolveService("250.1.1.1", 8333);
+    CService addr1 = ResolveService("250.1.1.1", (uint16_t)8333);
```

Comment:
Nit: Same here. I don't think this is worth the diff. You already changed the interface of `ResolveService()`, which should be enough, imo.